sqlglot/generator.py

```diff
@@ -3015,6 +3020,12 @@ def renamecolumn_sql(self, expression: exp.RenameColumn) -> str:
         new_column = self.sql(expression, "to")
         return f"RENAME COLUMN{exists} {old_column} TO {new_column}"
 
+    def alterset_sql(self, expression: exp.AlterSet) -> str:
+        exprs = self.expressions(expression, flat=True)
+        if not expression.find(exp.Properties):
+            exprs = f" {exprs}"
```

Comment:
This can be refactored if we clean up the `properties_sql` generation right?

Reply:
Yes, I added this in the PR feedback to offset the strip, but if `properties_sql` is fixed in that regard it won't be needed